create-a-simple-catalog-of-anything.md

```diff
@@ -0,0 +1,193 @@
+
```

Comment:
@elisapaka i think we are missing headings and frontmatter here ...

Reply:
@rufuspollock headings are in place as far as I can tell? They show up for me on the website: https://makeitmarkdown.flowershow.app/content/howtos/create-a-simple-catalog-of-anything

None of the pages have frontmatter as far as I'm aware (I'm understanding 'frontmatter' as a table at the top of each page which contains basic data about the file), but I can add it in if we want.